code/src/layouts/app-header/AppHeader.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import AppHeader from "@/layouts/app-header/AppHeader";
+import Header from "@/layouts/app-header/components/header/Header";
+import Categories from "@/layouts/app-header/components/categories/Categories";
+
+jest.mock("@/layouts/app-header/components/header/Header", () => () => (
+  <div>Header</div>
+));
+jest.mock("@/layouts/app-header/components/categories/Categories", () => () => (
+  <div>Categories</div>
+));
+
+test("renders Header and Categories components", () => {
+  const { getByText } = render(<AppHeader />);
+
+  expect(getByText("Header")).toBeInTheDocument();
+  expect(getByText("Categories")).toBeInTheDocument();

```

Comment:
What is the reason you use `getByText` from render and not `screen.getByText`?

Reply:
> What is the reason you use `getByText` from render and not `screen.getByText`?

I tried this case. I have the error

<img width="605" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/f3c46bf2-8517-4614-91af-165194b5545a">